SeleniumCookies/wrapper.py

```diff
@@ -348,6 +355,7 @@ def load(self):
 def create_cookie(host, path, secure, expires, name, value):
     """Shortcut function to create a cookie
     """
+    logging.debug('Creating Cookie')

```

Comment:
Use `logging.info` here

Reply:
lol, I was supposed not to use debug here and I end up using debug.

Reply:
No, If I use logging.info here then when level is set to info, it will flood the whole logfile with your cookies. Even if we set the level of logging to debug and have no logging.debug in code it will still flood the whole logfile. It's better not to go below info level and avoid using above debug in some functions which will be called rapidly(9940 times).